test/makeLatexReportRegression.m

```diff
@@ -0,0 +1,231 @@
+function makeLatexReportRegression(currentStatus, output, otherStatus)
+% generate a LaTeX report
```

Comment:
I want to see if this is useful and picks up use at all. 

We can already use it to inspect failed tets of existing PRs. The worklfow would be:

 -  Check out this PR
 -  run `regressionTest(commit to test against, commit of target PR, failed test indices)`